target/linux/ipq807x/files/arch/arm64/boot/dts/qcom/ipq8074-nbg7815.dts

```diff
@@ -0,0 +1,517 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/* Copyright (c) 2022, Karol Przybylski <itor@o2.pl> */
+
+/dts-v1/;
+
+#include "ipq8074.dtsi"
+#include "ipq8074-hk-cpu.dtsi"
+#include "ipq8074-ess.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	model = "Zyxel NBG7815";
+	compatible = "zyxel,nbg7815", "qcom,ipq8074";
+  
+	aliases {
+		serial0 = &blsp1_uart5;
+		/* LED is managed by lp5569
+    led-boot = &led_system_yellow;
+		led-failsafe = &led_system_yellow;
+		led-running = &led_system_blue;
+		led-upgrade = &led_system_yellow;
+    */
+		/* Aliases as required by u-boot to patch MAC addresses */
+		ethernet0 = &dp1;
+		ethernet1 = &dp2;
+		ethernet2 = &dp3;
+		ethernet3 = &dp4;
+		ethernet4 = &dp5;
+    ethernet5 = &dp6;
+		label-mac-device = &dp1;
+	};
+
+	chosen {
+    stdout-path = "serial0:115200n8";
+		bootargs = "console=ttyMSM0,115200n8 root=PARTUUID=a0bb3852-36cc-ac69-a1a2-6bd39ff56f0a rootwait";
+		bootargs-append = " swiotlb=1";
+	};
+  
+  
+  keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			gpios = <&tlmm 36 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+};
+
+&tlmm {
+  mdio_pins: mdio-pins {
+    mdc {
+      pins = "gpio68";
+      function = "mdc";
+      drive-strength = <8>;
+      bias-pull-up;
+    };
+
+    mdio {
+      pins = "gpio69";
+      function = "mdio";
+      drive-strength = <8>;
+      bias-pull-up;
+    };
+  };
+};
+
+&blsp1_uart5 {
+  status = "okay";
+};
+
+&prng {
+  status = "okay";
+};
+
+&cryptobam {
+  status = "okay";
+};
+
+&crypto {
+  status = "okay";
+};
+
+&qpic_bam {
+  status = "okay";
+  };
+
+&mdio {
+	status = "okay";
+
+	pinctrl-0 = <&mdio_pins>;
+	pinctrl-names = "default";
+	reset-gpios = <&tlmm 37 GPIO_ACTIVE_LOW>; /*<0x14 0x25 0x00 0x14 0x1f 0x01>;*/
+
+	qca8075_0: ethernet-phy@0 {
+		reg = <0>;
+	};
+
+	qca8075_1: ethernet-phy@1 {
+		reg = <1>;
+	};
+
+	qca8075_2: ethernet-phy@2 {
+		reg = <2>;
+	};
+
+	qca8075_3: ethernet-phy@3 {
+		reg = <3>;
+	};
+
+	qca8075_4: ethernet-phy@4 {
+		reg = <0x1c>;

```

Comment:
Is this the 2.5G PHY?

If so, label is completely misleading and you are missing its reset GPIO